orangecontrib/timeseries/datasets/daydt.csv

```diff
@@ -0,0 +1,9 @@
+Day,Value
```

Comment:
_orangecontrib.timeseries.datasets_ was meant for public toy timeseries data sets, not unit test data. :thinking:

Reply:
Any idea where I should place these, bar the python file itself?

Reply:
As they are fairly miniature, I personally have no reservations about seeing them in the test file itself. Otherwise somewhere under _tests_ seems appropriate.

Reply:
FWIC, with [biolab/datasets](https://github.com/biolab/datasets) more widely adopted, I have no issue with repurposing/moving the _datasets_ dir too ...

Reply:
I agree that it doesn't make sense to include the files in the same folder as a real dataset. Thanks, I've moved it.